lib/system/generic/xlnx_common/zynqmp_aarch64/sys.c

```diff
@@ -27,6 +27,9 @@
 #include "xreg_cortexa53.h"
 #endif /* defined(versal) */
 
+#define MB (1024 * 1024UL)
```

Comment:
Thanks @arnopo for this change.
Just to avoid any future warnings, we can do "#ifndef MB".
But, me or @bentheredonethat will do that change.

Reply:
done